packages/web/src/components/gcds-nav-link/stories/gcds-nav-link.stories.tsx

```diff
@@ -0,0 +1,62 @@
+export default {
+  title: 'Components/Navigation link',
+
+  argTypes: {
+    // Props
+    href: {
+      name: 'href',
+      control: 'text',
+      table: {
+        type: { summary: 'string' },
+        defaultValue: { summary: '-' }
+      },
+      type: {
+        required: true
+      }
+    },
+    current: {
+      name: 'current',
+      control: 'boolean',
+      table: {
+        type: { summary: 'boolean' },
+        defaultValue: { summary: false }
+      },
+    },
+
+    // Slots
+    default: {
+      control: {
+        type: 'text',
+      },
+      table: {
+        category: 'Slots | Fentes',
+      }
+    },
+  },
+};
+
+const Template = (args) => (`
+// Web Component (Angular, Vue)
+<gcds-nav-link
+  href="${args.href}"
+  ${args.current ? `current` : null}
+>
+  ${args.default}
+</gcds-nav-link>
+
+
+// React code
```

Comment:
Can we update this comment to `<!-- React code -->` so that it doesn't show in the preview?